RecoPixelVertexing/PixelVertexFinding/src/gpuVertexFinder.h

```diff
@@ -66,6 +74,7 @@ namespace gpuVertexFinder {
 	     float ichi2max,   // max normalized distance to cluster
              bool ienableTransfer
 	     ) :
+      onGPU{nullptr},
```

Comment:
Would you mind if we use `onGPU(nullptr)` for consistency with the rest ?

I don't have a strong opinion though, I'll merge it like this if you prefer.